src/__tests__/fixtures/config-extending-override.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file shouldn't be named `config-extending-override` because it's not extending anything. How about `config-string-quotes-single.json`? (Matching convention of `config-block-no-empty`, etc.)